src/routes/github/configuration/github-configuration-post.ts

```diff
@@ -64,7 +63,7 @@ export const GithubConfigurationPost = async (req: Request, res: Response): Prom
 		}
 
 		const subscription = await Subscription.install({
-			clientKey: getHashedKey(req.body.clientKey),
+			clientKey: req.body.clientKey,
```

Comment:
Shouldn't it be our next step? I thought first we will be writing both keys in parallel? Can we escape from downtime without it?

Reply:
Because admin api need to update the plain client key
  ---> we don't want to do migration again and again
     --->So the normal `install` method must save plain key to new col
         --> `Subscription.install` 's arg is hashed, so need to change it
           --> :)